src/index.html

```diff
@@ -1,13 +1,413 @@
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-action">
+            <a href="/" class="logo">
+              <img
+                class="logo__img"
+                src="images/logo_menu.png"
+                alt="Take the Streets"
+              >
+            </a>
+
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">
+                Home
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about" class="menu__link">
+                About us
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#bikes" class="menu__link">
+                Compare Bikes
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">
+                Details
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">
+                Contacts
+              </a>
+            </li>
+          </ul>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__call menu__call--phone">
+            +1 234 5555-55-55
+          </a>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__call menu__call--action">
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main class="main">
+      <section class="main__section about" id="about">
+        <div class="container">
+          <div class="about__content">
+            <h2 class="main__section-title about__title">
+              Move Free
+            </h2>
+
+            <article class="about__paragraph">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </article>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__section bikes" id="bikes">
+        <div class="container">
+          <h2 class="main__section-title bikes__title">
+            Compare Bikes
+          </h2>
+
+          <div class="bikes--container">
+            <div class="bikes__item">
+              <div class="bikes__photo">
+                <img
+                  src="images/photos/bike/1.png"
+                  alt="Sporty 4"
+                  class="bikes__photo-img"
+                >
+                </img>
+              </div>
+
+              <h3 class="bikes__name">
+                Sporty 4
+              </h3>
+
+              <p class="bikes__about">
+                The iconic frame brought to a new performance height as a sporty, active ride.
+              </p>
+
+              <div class="bikes__prise">
+                $ 2 590
+              </div>
```

Comment:
Price text should be bold
![image](https://user-images.githubusercontent.com/37161315/170648455-54f3e012-14e3-4d45-813e-6f26b6a9329f.png)